src/mbgl/tile/tile_loader_impl.hpp

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include <mbgl/tile/tile_loader.hpp>
-#include <mbgl/storage/file_source.hpp>
 #include <mbgl/renderer/tile_parameters.hpp>
+#include <mbgl/storage/file_source.hpp>
+#include <mbgl/tile/tile_loader.hpp>
+#include <mbgl/util/async_request.hpp>
 #include <mbgl/util/tileset.hpp>
 
 #include <cassert>
 
+constexpr const char* CANT_LOAD_TILE = "Can't load tile.";
```

Comment:
nit: you could make the whole `std::make_exception_ptr(std::runtime_error("Can't load tile"))` a constexpr function like `getCantLoadError()`.